ADAL/src/cache/ADTokenCache.m

```diff
@@ -567,7 +603,7 @@ - (BOOL)addOrUpdateImpl:(ADTokenCacheItem *)item
 
 - (NSString *)description
 {
-    return [NSString stringWithFormat:@"ADTokenCache: %@", _cache];
+    return [NSString stringWithFormat:@"ADTokenCache: %@", [self cacheCopy]];
```

Comment:
I don't understand. -cacheCopy returns an autoreleased copy of the _cache object, which should be completely safe to use here. I'd hate to have to add a @synchronize call here if I don't have to.